src/Sylius/Component/Order/Model/Order.php

```diff
@@ -491,7 +504,7 @@ public function getAdjustmentsTotal($type = null)
         }
 
         $total = 0;
-        foreach ($this->getAdjustments($type) as $adjustment) {
+        foreach ($this->getAdjustmentsRecursively($type) as $adjustment) {
```

Comment:
Why? `getAdjustmentsTotal` should return only adjustments on current level, do we need to get it recursively anywhere?